integreat_cms/static/src/css/contact_card.css

```diff
@@ -0,0 +1,26 @@
+.contact-card {
+    display: inline-block;
+    box-sizing: border-box;
+    padding: 1rem;
+    border-radius: 0.25rem;
+    background-repeat: no-repeat;
+    background-color: rgba(127, 127, 127, 0.15) !important;
+    background-image: linear-gradient(to right, rgba(255, 255, 255, 0.9) 0 100%), url("../svg/contact.svg") !important;
+    background-blend-mode: difference;
+    background-position: calc(100% + 2em) calc(100% + 1em);
+    background-size: 7em;
+    box-shadow: 0 0.1em 0.1em rgba(0, 0, 0, 0.4);
+    cursor: default !important;
+    color: initial;
+    text-decoration: initial;
+    min-width: 50%;
+    white-space: initial !important;
+
+    p {
+        margin: 4px 0 0 0;
+    }
+
+    h4 {
+        margin: 0 0 12px 0;
+    }
+}
```

Comment:
```suggestion
.contact-card {
    display: inline-block;
    box-sizing: border-box;
    padding: 1rem;
    border-radius: 0.25rem;
    background-repeat: no-repeat;
    background-color: rgba(127, 127, 127, 0.15);
    background-image: linear-gradient(to right, rgba(255, 255, 255, 0.9) 0 100%), url("../svg/contact.svg");
    background-blend-mode: difference;
    background-position: calc(100% + 2em) calc(100% + 1em);
    background-size: 7em;
    box-shadow: 0 0.1em 0.1em rgba(0, 0, 0, 0.4);
    color: initial;
    text-decoration: initial;
    min-width: 50%;
    white-space: initial;

    p {
        margin: 4px 0 0 0;
    }

    h4 {
        margin: 0 0 12px 0;
    }
}
```

I would like to avoid using `!important` selectors if possible. In this case, I think we can make the `[translate="no"]` selector ignore contact cards, and move the `cursor: default !important;` to `tinymce_custom.css`